### PR TITLE
Polls 2022 01 06

### PR DESCRIPTION
**Pull request by TheExistence**
## Summary of changes

_Description of the changes included in this pull request. 
Steps to follow to confirm the feature is completed or the bug/issue is resolved. 
Are there any dependencies? 
Does this code break any existing functionality or introduce any risk?_

## Relevant Context

- Feature: [ticket number](link to project management ticket)
- Relevant Pull Request: # 

## Further comments

--- 
## Types of changes

- [ ] Content edit, File upload, content addition
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Technical Documentation Update
- [ ] Other (please describe):

## Does this introduce a breaking change?

_Breaking change is defined as_ **"A fix or feature that would cause existing functionality to change."**

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added/changed necessary documentation (if appropriate)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: governance/polls/MIP9 Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022
+summary: Signal your support or opposition to prioritising onboarding TUSD (TrueUSD).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022.md:3]: Can we make it clear that the application is for a PSM - see the forum thread

File: governance/polls/MIP9 Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-01-10T16:00:00
+end_date: 2022-01-24T16:00:00
+---
+# Poll: Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022.md:16]: Can we make it clear that the application is for a PSM - see the forum thread

File: governance/polls/MIP9 Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, January 10 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset TUSD (TrueUSD) should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - TUSD (TrueUSD) - January 10, 2022.md:27]: Can we make it clear that the application is for a PSM - see the forum thread

**Review by mkrorbrkr — CHANGES_REQUESTED** [review]
Looks pretty good! See comments about the TUSD poll

File: governance/polls/MIP9 Community Greenlight Poll - TUSD-PSM (TrueUSD) - January 10, 2022.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, January 10 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether applying a PSM using TUSD as collateral should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - TUSD-PSM (TrueUSD) - January 10, 2022.md:27]: Could you change "applying to adding" and then I think it's good to go after that

**Review by mkrorbrkr — APPROVED** [review]
All looks good to me @prose11

**Review by prose11 — APPROVED** [review]
Nice work :)

**Pull request merged**